notifier.rst

```diff
@@ -55,6 +55,7 @@ with a couple popular SMS services:
 ==============  ====================================  ===========================================================================
 Service         Package                               DSN
 ==============  ====================================  ===========================================================================
+46elks          ``symfony/46elks-notifier``           ``46elks://API_USERNAME:API_PASSWORD@default?from=FROM``
```

Comment:
You also need to add a versionadded directive (see same file on v5.4 branch to have hint :))

Reply:
Versionadded added